src/main/java/com/exasol/release/GitHubReleaseMaker.java

```diff
@@ -0,0 +1,33 @@
+package com.exasol.release;
+
+import java.util.logging.Logger;
+
+import com.exasol.github.GitHubRepository;
+
+/**
+ * This class responds for releases on GitHub
+ */
+public class GitHubReleaseMaker implements ReleaseMaker {
+    private static final Logger LOGGER = Logger.getLogger(GitHubReleaseMaker.class.getName());
+    private final GitHubRepository repository;
+
+    /**
+     * Create a new {@link GitHubReleaseMaker}.
+     * 
+     * @param repository an instance of {@link GitHubRepository}
+     */
+    public GitHubReleaseMaker(final GitHubRepository repository) {
+        this.repository = repository;
+    }
+
+    @Override
+    public void makeRelease() {
+        LOGGER.fine("Releasing on GitHub.");
+        final String version = this.repository.getVersion();
+        final String changes = this.repository.getChangesFile(version);
+        final int firstLineEnd = changes.indexOf('\n');
+        final String releaseName = changes.substring(0, firstLineEnd);
+        final String releaseLetter = changes.substring(firstLineEnd + 1);
```

Comment:
Extract methods `readReleaseNameFromReleaseLetter` and `readReleaseContentFrom ReleaseLetter`.

Also this code probably ends in a substring out-ouf-bounds error if the new line is missing or if no more characters come after the newline. Please add test cases for this.

Reply:
I'm going to adjust this part, but on a new branch